Description: Exclude the NDEBUG macro definition
 This macro is not used directly by Telegram Desktop but affects WebRTC headers
 that were built without it. The macro affects internal classes size that may
 lead to memory corruption and crashes. If someone still wants to define the
 _DEBUG macro, they can do so via the DEB_CPPFLAGS_APPEND environment variable.
Forwarded: not-needed
Author: Nicholas Guriev <guriev-ns@ya.ru>
Last-Update: Sun, 07 Feb 2021 11:10:09 +0300

--- a/cmake/options.cmake
+++ b/cmake/options.cmake
@@ -9,7 +9,6 @@ add_library(desktop-app::common_options
 
 target_compile_definitions(common_options
 INTERFACE
-    $<IF:$<CONFIG:Debug>,_DEBUG,NDEBUG>
     QT_NO_CAST_FROM_BYTEARRAY
 )
 
